addons/website/static/src/snippets/s_banner/options.js

```diff
@@ -0,0 +1,14 @@
+odoo.define("website.s_banner_options", function (require) {
+    "use strict";
+
+    const options = require("web_editor.snippets.options");
+
+    options.registry.Banner = options.Class.extend({
+        start: function () {
+            // TODO adapt in master. s_nb_column_fixed class is to be removed
+            // in XML as it disabled the column options for banner snippet.
```

Comment:
Not sure but I think that a migration script will be needed for master too :thinking: Since this "patch" here will be removed, the previously dropped snippets that would not have been started again would still have the class :slightly_smiling_face: